support-frontend/assets/components/subscriptionCheckouts/payPalSubmitButton.jsx

```diff
@@ -84,17 +65,12 @@ function SubscriptionSubmitButtons(props: PropTypes) {
             billingPeriod={props.billingPeriod}
           />
         </div>
-        <Button
-          id="qa-submit-button"
-          type="submit"
-          modifierClasses={props.paymentMethod === PayPal ? ['hidden'] : []}
-        >
-          Continue to payment
-        </Button>
       </div>
-      <span>{props.allErrors.length > 0 && <ErrorSummary errors={props.allErrors} />}</span>
+      {props.paymentMethod === PayPal && (
```

Comment:
it is a bit funny that we have to replicate the errors in both buttons.  it would be better if we can split the errors somewhere that's common to both, as I think the errors are unrelated to which kind of button paypal or other we want to use

Reply:
The difficulty that we have here is that the errors belong to the fields, and in following the pattern of keeping the implementation in the component, we need to pass the errors through to the button that's being used. Happy to discuss if this doesn't make sense.

Reply:
discussed in person, sounds ok
The reason is it's because the aggregation of errors is actually implemented as a validation error on the "submit field" so it has to be contained in the same div.  And anyway most of the implementation is shared in the ErrorSummary component.